exporter/clients.go

```diff
@@ -44,32 +55,100 @@ func parseClientListString(clientInfo string) (*ClientInfo, bool) {
 		}
 
 		switch vPart[0] {
+		case "id":
+			connectedClient.Id = vPart[1]
 		case "name":
 			connectedClient.Name = vPart[1]
 		case "user":
 			connectedClient.User = vPart[1]
 		case "age":
 			createdAt, err := durationFieldToTimestamp(vPart[1])
 			if err != nil {
-				log.Debugf("cloud not parse age field(%s): %s", vPart[1], err.Error())
+				log.Debugf("could not parse 'age' field(%s): %s", vPart[1], err.Error())
 				return nil, false
 			}
 			connectedClient.CreatedAt = createdAt
 		case "idle":
 			idleSinceTs, err := durationFieldToTimestamp(vPart[1])
 			if err != nil {
-				log.Debugf("cloud not parse idle field(%s): %s", vPart[1], err.Error())
+				log.Debugf("could not parse 'idle' field(%s): %s", vPart[1], err.Error())
 				return nil, false
 			}
 			connectedClient.IdleSince = idleSinceTs
 		case "flags":
 			connectedClient.Flags = vPart[1]
 		case "db":
 			connectedClient.Db = vPart[1]
+		case "sub":
+			Sub, err := strconv.ParseInt(vPart[1], 10, 64)
+			if err != nil {
+				log.Debugf("could not parse 'sub' field(%s): %s", vPart[1], err.Error())
+				return nil, false
+			}
+			connectedClient.Sub = Sub
+		case "psub":
+			Psub, err := strconv.ParseInt(vPart[1], 10, 64)
```

Comment:
I think we can simplify all these a lot by ignoring the parse error and simply do

`connectedClient.Psub, _ = strconv.ParseInt(vPart[1], 10, 64)`

(and can you maybe use Atoi instead ? is there an explicit need for the uint64?)

Can you just add a little to the tests that the metrics are not only present but have some none-zero value?

Reply:
think we can simplify all these a lot by ignoring the parse error and simply do

connectedClient.Psub, _ = strconv.ParseInt(vPart[1], 10, 64)

(and can you maybe use Atoi instead ? is there an explicit need for the uint64?)
Atoi returns a native int - whose size varies between platforms (32 bit in x86 & arm). Usage of it seems like an avoidable risk. The above was lifted from the previous code.

Can you just add a little to the tests that the metrics are not only present but have some none-zero value?
Yep, added!

Reply:
> Atoi returns a native int - whose size varies between platforms (32 bit in x86 & arm). Usage of it seems like an avoidable risk. The above was lifted from the previous code.

what's the risk? 

I don't see 'ParseInt()` used other than in `durationFieldToTimestamp()` (where it's not really needed IMO) so let's keep it simple and use Atoi

Reply:
If I may interject...

The risk is integer overflow. At least for some of the fields (e.g. the memory & buffer size fields), we should use an integer type that guarantees 64-bits, since that's what Redis code uses: https://github.com/redis/redis/blob/7.4.0/src/networking.c#L2878

My understanding is that Atoi casts to an int which only guarantees 32 bits.

Reply:
Fair point @hiroshi-ishii - let's keep the 64 bit ints

Reply:
applied simplified parsing.